docker/README.md

```diff
@@ -4,11 +4,18 @@ Temporal builds 4 Docker images with [every release](https://github.com/temporal
 
 ## Prerequisites
 
-To build docker image:
-  * [docker](https://docs.docker.com/engine/installation/)
+To build docker image, you will need [Docker](https://docs.docker.com/engine/installation/).
 
-To run docker image with dependencies:
-  * [docker-compose](https://docs.docker.com/compose/install/)
+To *run* the docker image with dependencies (e.g. PostgreSQL and Elasticsearch), you can use `docker-compose`. See our [docker-compose repo](https://github.com/temporalio/docker-compose) for reference configurations.
```

Comment:
There is still a missing piece to use the locally built images.

The tags in the docker compose need to be updated to `test`

Assuming that is the goal of the update, build locally and make use of those built images.

Reply:
On this topic, I think its time to add a `.env` file to the docker-compose repo.

It would be simpler to override the tag and for version updates as well.